### DISCUSSION TITLE
The curious case of Asus Predator XB271HU

### DISCUSSION THREAD

**Discussion by Sputgnocchi** [xanderfrangos/twinkle-tray#738] [Monitor Compatibility Issues]
Hello everyone,

I'm encountering an issue with the Asus Predator XB271HU monitor. Despite there being approximately 4 variants of this model released, they should all function similarly. However, I'm experiencing unresponsiveness when attempting to adjust the brightness. Twinkle recognizes the current brightness set through monitor commands, confirming DDC/CI compatibility, but software adjustments aren't effective.

Asus, as always, are a terrible company when it comes to driver support, customer support, or anything beyond the BS marketing and distribution to put their products on any shelf of a big retailer. If there's anything **"Predator"y** in their product line, it's their support.

![image](https://github.com/xanderfrangos/twinkle-tray/assets/76185682/857302ac-ccac-4123-9740-5f7dad16b75a)

**Comment by Sputgnocchi** [comment]
ps. I also tried different variations of the troubleshooting options in the program menu, in the stable and beta programs. And the monitor doesn't have any DDC/CI settings available in the menu.

**Comment by xanderfrangos** [comment]
Yeah, this is unfortunately not the first time this monitor has come up. Does this help? https://github.com/xanderfrangos/twinkle-tray/issues/461#issuecomment-1304919755

**Reply by Sputgnocchi** [reply]
Thanks my friend.
You're right, it was brought up before and I missed it. My apologies.
____

In return, if anyone else miss this here's how I'm managing the twinkle implementation on this monitor:

1) Turn on the monitor by holding the gaming "G" button - it will enter the "factory mode" and enable the "Burn In Pattern" (which is enabled by default when you enter factory mode)
2) By only doing step 1, you will be able to manage the brightness using a DDC/CI software
3) The issue now is that your monitor will be flashing RED/BLUE/GREEN every time it goes into sleep mode or it doesn't receive any signal from DP/HDMI - solution: turn off the monitor when you step away or turn off the computer
